tests/bluetooth/bsim/audio/src/vcp_vol_ctlr_test.c

```diff
@@ -273,452 +242,968 @@ static void connected(struct bt_conn *conn, uint8_t err)
 	g_is_connected = true;
 }
 
-static void bt_ready(int err)
+BT_CONN_CB_DEFINE(conn_callbacks) = {
+	.connected = connected,
```

Comment:
It is out of scope for this PR, but it should be possible to move the 'connected' callback also to the common.c file, just as has been done with the 'disconnected' callback

Reply:
Yes and no: The `connected` callbacks may differ from audio test to audio test, so it's not always the same we can use. 

What we should do is define a common `connected` that most of the tests can use (and find a way to handle the flags set in it), and then have specific connected callbacks to those that need additional handling. 

We should also dynamically register them, rather than use `BT_CONN_CB_DEFINE` as explained by my BSIM presentation a while ago :) 

But both changes are minor changes that does change the behavior of existing tests, and thus have a fairly low priority